codec-registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/ClassSweepCompilerPluginComponent.scala

```diff
@@ -20,7 +20,7 @@ class ClassSweepCompilerPluginComponent(options: CodecRegistrationCheckerOptions
 
   override def newPhase(prev: Phase): Phase =
     new StdPhase(prev) {
-      private val up = options.oldTypes.groupBy(_._2)
+      private val oldTypesFromPreviousCompilation = options.oldTypes.groupBy(_._2)
```

Comment:
:benissimo:

Reply:
I'd decompose the tuple explicitly here, to avoid `_2`

Reply:
this has been fixed by switching from Tuple2 to a case class in the whole module - see:
case class ParentChildFullyQualifiedClassNamePair
inside the CodecRegistrationCheckerOptions.scala file